index.ts

```diff
@@ -1,4 +1,4 @@
-/// <reference path='typings/tsd.d.ts' />
+/// <reference path='typings/definitions.d.ts' />
```

Comment:
Since `index.ts` is outside lib folder, should it be `lib/typings/definitions.d.ts`?